src/components/CryptoCoinItem/cryptoAPI.ts

```diff
@@ -0,0 +1,12 @@
+import {CryptoApiResponse} from '../types';
+
+export const fetchCryptoData = async () => {
+  try {
+    const response = await fetch('https://data.messari.io/api/v1/assets');
```

Comment:
[critical] The conecction with the api is outside the scope, it should be implemented in the correct PR (the second )

Reply:
use mocked data